### PR TITLE
updated according to requested changes

### PR DESCRIPTION
**Pull request by sparshmanni**

### CODE REVIEW

File: src/count_occurrence.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+  return count;
+};
+```
+
+### C++
+
+```c++
+int countOccurrences(const std::vector<int>& arr, int valueToCount) {
```
Comment (rahulrk-dev) [inline src/count_occurrence.md:45]: See if u can do it without including vector lib

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]

**Comment by rahulrk-dev** [comment]
Rename file in camel case (countOccurrence.md)

**Comment by sparshmanni** [comment]
> Rename file in camel case (countOccurrence.md)   -DONE

> Removed vector from C++ snippet  -DONE

File: src/countOccurrence.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+  return count;
+};
+```
+
+### C++
+
+```c++
+int countOccurrences(int arr[], int size, int valueToCount) {
```
Comment (rahulrk-dev) [inline src/countOccurrence.md:45]: Why passing 3 arguments here, it's changing the snip structure

Comment (rahulrk-dev) [reply src/countOccurrence.md:45]: If it's getting too complex, you can remove snip for snip for C++

File: src/countOccurrence.md
Comment (sparshmanni) [inline src/countOccurrence.md:1]: Removed third para from function for C++

**Review by rahulrk-dev — APPROVED** [review]
Thanks @sparshmanni, merging

**Pull request merged**